docs/Releasing.md

```diff
@@ -4,16 +4,38 @@ Open Enclave SDK Release Procedures
 This document covers how we handle creating a release. Let's look at it from an
 example:
 
+Selecting a Release Manager
+---------------------------
+
+A committer can be nominated to become the release manager by drafting a pull
+request that sets the "Release" tag to the "Area" of one of the committers in
+[Committers.md](Committers.md). The submitter of that nomination should then
+send a link to that pull request through email to all of the committers for open
+discussion. This discussion & PR should be open for at least 3 days, and people
+can make their votes known in the pull request itself by either approving the
+pull request or requesting changes. If there is any opposition from the
+committers after 3 days, then the Committee Governance Committee should take up
+this issue as high priority over email and decide whether to accept or reject
+the new release manager based on feedback from the other committers. If there is
+no opposition from the committers after three days, the newly nominated
+committer becomes the release manager and the pull request to add the "Release"
+tag to this committer gets merged to master. Only committers can nominate other
+committers to be the release manager. Only one committer can be tagged with the
+Release tag in [Committers.md](Committers.md).
+
 Version Bump
 ------------
 
+*Note*: The versioning information below is not fully applicable until it can be
+updated. There were a few issues with both Debian and NuGet packages regarding
+the v0.7.x naming scheme. This process will be changing soon. The release branch
```

Comment:
That naming scheme shouldn't ever make it into a package. The `x` explicitly denotes that it's an unreleased branch (i.e. _not_ a version). If a package is being generated, it should go with whatever the current patch is (I guess 0 if there is no v0.7 release) and then potentially iterate with RC suffixes.

Reply:
I personally don't like having a version set in the VERSION file before that version actually exists...but I'm not sure of another way to do this.

Reply:
The problem with using `x` will surface if you build a package on the master branch. It simply will not function with NuGet packages. Moreover, 0.7.x is considered a greater version than 0.7.0 for debian packages, and that is arguably even more broken because you won't even be aware that a package is available for upgrade when 0.7.0 is published to an apt repo. It's just not a good versioning scheme.

I proposed a possible different way of structuring the VERSION file in #2204: 

```
Just rebased this branch on master. I think I've addressed all comments in this PR. 
There are a few open comments to welcome discussion. Specifically, the VERSION file and how NuGet handles it is a little awkward.
I think we may wish to redesign the VERSION file to be something like:

OE_MAJOR=0
OE_MINOR=7
OE_PATCH=0
OE_PRERELEASE=alpha
OE_BUILD=1

as per https://semver.org/

Then, we can parse this file to produce a package's version number, depending on the type of package.
Different package formats (RPM, DEB, NuGet, etc) expect different version strings, and this kind of VERSION file makes it easier to parse for translation into a full version string for a given package format.
I'd like to address this in a future design doc.
```

Reply:
We probably don't want to get into the situation where we're updating this version file with every commit, because that produces a lot of commit message churn (and it was very annoying). But at least with this kind of situation we have a prerelease variable we can set to anything we want it to be. In the master branch it could be `dev`, and in the v0.7.x branch we could update that variable for `rc` or something, until its actually released, and then change it to `release` or something.